roles/firewall/tasks/main.yml

```diff
@@ -71,6 +71,13 @@
   tags:
     - firewall
     - monitoring
+    
+- name: Allow ZMQ connections
+  ufw:  rule=allow
+        port=5550
+        proto=tcp
```

Comment:
Security concern: can we add a domain/IP/something here to limit where we accept the connections on that port from?

Is there only going to be 1 server (validation.l.o) that will push the messages down to kernelci?